CMake/ITKSetPython3Vars.cmake

```diff
@@ -36,18 +37,16 @@ if("${CMAKE_VERSION}" VERSION_LESS_EQUAL "3.12.0")
     set(Python3_LIBRARIES    ${PYTHON_LIBRARIES})
   endif()
 else()
+  cmake_policy(SET CMP0094 NEW) # makes FindPython3 prefer activate virtualenv Python to latest version
   if(PYTHON_DEVELOPMENT_REQUIRED)
-    if(DEFINED Python3_EXECUTABLE)
-      set(_specified_Python3_EXECUTABLE ${Python3_EXECUTABLE})
-    endif()
     find_package(Python3 COMPONENTS Interpreter Development)
-    set(Python3_EXECUTABLE ${_specified_Python3_EXECUTABLE} CACHE INTERNAL
-      "Path to the Python interpreter" FORCE)
   else()
     find_package(Python3 COMPONENTS Interpreter)
   endif()
-  if(NOT Python3_EXECUTABLE AND _Python3_EXECUTABLE)
-    set(Python3_EXECUTABLE ${_Python3_EXECUTABLE} CACHE INTERNAL
-      "Path to the Python interpreter" FORCE)
-  endif()
 endif()
+
+get_cmake_property(_variableNames VARIABLES)
+list (SORT _variableNames)
+foreach (_variableName ${_variableNames})
+    message(STATUS "${_variableName}=${${_variableName}}")
+endforeach()
```

Comment:
remove debug code

Reply:
I'm not sure the best way to do what I'm doing.  I currently debugging the .YAML code by amending my commit.  I'm trying to debug the WindowsPython Azure pipeline.  It still doesn't work, even with the above changes.  So the next couple of commits are going to be me debugging and running WindowsPython on my personal Azure account.  Just going off the log, I have no idea how it's finding and specifying Python 3.9.0 (it's not in any of the environment variables).

Reply:
Not to get ahead of myself, but I had a typo in the registry variable ```Python_``` vs ```Python3_```.  I'm pretty sure it was searching the registry.  Hopefully good news tomorrow.

Reply:
Got it working, this should fix the WindowsPython pipeline crashing.